### ISSUE TITLE
Inverted logic of access control permission "no-monitoring/​contacts" in module monitoring breaks inheritance

### ISSUE THREAD

**Issue by domme0711** [Icinga/icingaweb2#4644]
## Describe the bug
The permission "no-monitoring/​contacts" of the monitoring module works the wrong way, which causes problems in role inheritance.
There is no deny-switch for this permission.
The logic how all other permissions work is inverted here.

## To Reproduce
1. Add a role "some-basic-role" and allow the permission "no-monitoring/​contacts".
2. Add a second role "some-advanced-role" and let it inherit from the first one.
3. Try to let a user of "some-advanced-role" see any contact. There is no deny-switch for that inverted permission and no way to allow accessing contacts via UI for this role. It is only possible by adding this permission as a refusal to this role in roles.ini, but this is intransparent to the UI.

## Expected behavior
It shoult be possible that users of "some-advanced-role" can see contacts while users of role "some-basic-role" can not.

Either provide the deny-switch button also to this permission, or
invert the whole thing (remove the "no-" part).
Permission shoult be called "monitoring/contacts" instead of "no-monitoring/contacts".

## Your Environment
Icingaweb2 v2.9.5

**Comment by nilmerg** [comment]
> It shoult be possible that users of "some-advanced-role" can see contacts while users of role "some-basic-role" can not.

No, absolutely not. Refusals override grants, and since this permission is inverted, it is effectively a refusal and "granting" it cannot be negated again. That's exactly the reason why it is not possible to refuse in the UI.
If you refuse a grant in "some-basic-role", it has the exact same effect.

> Permission shoult be called "monitoring/contacts" instead of "no-monitoring/contacts".

This permission got introduced at a time when there was no support for actual refusals. When refusals got eventually introduced, we decided not to change this permission's meaning in order to not break user configurations.

If you want "some-advanced-role" to be able to access something, don't let it inherit a role that refuses it. Be it the refusal to schedule downtimes or the `no-monitoring/contacts` fake refusal.
